src/app/template-library/template-experience-card/template-experience-card.component.html

```diff
@@ -0,0 +1,8 @@
+<ion-card class="cursor-pointer">
+  <div *ngIf="template.leadImageUrl" class="card-media" [style.background-image]="'url(' + template.leadImageUrl + ')'"></div>
+  <div *ngIf="!template.leadImageUrl" class="card-media" [style.background-image]="'url(/assets/exp-placeholder.png)'"></div>
```

Comment:
same as the other component. we can combine those two lines into one

Reply:
This can be a simple `<img>` as well

Reply:
I'm doing it as a background of a div so that we don't need to worry about the dimension of the image used. This way no matter the dimension of the image used as the lead image it will always fit this div.
As far as I'm aware it's very hard to control this if you are using an `<img>` tag. Happy to update it though if you have a nice way of doing it